jenkins/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:16.04
+
+RUN apt-get update
+RUN apt-get install -y curl software-properties-common git
+RUN curl -fsSL get.docker.com | sh
+
+##
+## Go
+##
+RUN add-apt-repository ppa:longsleep/golang-backports
+RUN apt-get update
+RUN apt-get install -y golang-go
```

Comment:
Why not `FROM` golang?

Reply:
let me try

Reply:
Install mongodb server `RUN apt-get install -y mongodb-org` failed in golang image.
It seems to be  a dependency issue with libssl.

Reply:
Alright, thank you